### ISSUE TITLE
"Series" field needs to be made multiple in the "all fields" sample json in the guide

### ISSUE THREAD

**Issue by landreev** [IQSS/dataverse#9633]
Note: this needs to be done before 5.14!

The field "series" has been made multiple since 5.13. However, the example "all fields" json file we provide in the guide (`scripts/api/data/dataset-create-new-all-default-fields.json`) still has a single-value entry for it, making the json unimportable, once the soc. sci. metadata block is updated: 
```
            "typeName": "series",
            "multiple": false,
            "typeClass": "compound",
            "value": {
              "seriesName": {
              ...
```
It just needs to be updated the same way we've already updated the file `src/test/java/edu/harvard/iq/dataverse/export/ddi/dataset-create-new-all-ddi-fields.json` (used by the ddi export tests): 
```
           "typeName": "series",
            "multiple": true,
            "typeClass": "compound",
            "value": [ 
              {
                ...
```

Worth noting that we've been through the same issue with the other field that was recently made multiple - productionPlace. Plus there were all sorts of other things that were missed when those 2 fields were updated in their respective blocks that we had to keep fixing. I will open an issue for documenting a checklist of all such things and all the places where things need to be changed. So we can make it 100% the responsibility of the developer, if (God forbid) anyone else ever needs to make another existing single field multiple. (edit: opened #9634)

**Comment by cmbz** [comment]
Moved item to Needs Sizing because issue must be resolved for 5.14 release.

**Comment by poikilotherm** [comment]
Please note: there is already a PR and it's sized.
